### DISCUSSION TITLE
;2u escape sequence

### DISCUSSION THREAD

**Discussion by 9mm** [neovide/neovide#2031] [Q&A] [answered]
Hey, I have installed `floaterm` and I noticed typing something in caps like `SHIFT ENTER` will yield `SHIFT;2uENTER`

I tried in neovim itself and it didnt seem to occur. it only happens in a floating window though not normally

Also I did try searching but `;2u` isnt really easy to search hopefully this isnt a duplicate

**Comment by fredizzimo** [comment, marked as answer]
This is actually a Neovim bug https://github.com/neovim/neovim/issues/24093. You can fix it by remapping.

**Reply by 9mm** [reply]
Great thank you!

```
-- fix neovim escape sequences in terminal https://github.com/neovim/neovim/issues/24093
vim.keymap.set('t', '<S-Space>', ' ')
vim.keymap.set('t', '<S-CR>', '<CR>')
```
